plugins/pushbullet.js

```diff
@@ -1,88 +1,186 @@
 /**
  * Created by rocketman1337345 on 8/14/16.
+ * Re-written by werkkrew on 1/25/2018
  */
 
-var pushbullet = require("pushbullet");
-var _ = require('lodash');
-var log = require('../core/log.js');
-var util = require('../core/util.js');
-var config = util.getConfig();
-var pushbulletConfig = config.pushbullet;
+const pushbullet = require("pushbullet");
+const _ = require('lodash');
+const moment = require('moment');
+const log = require('../core/log.js');
+const util = require('../core/util.js');
+const config = util.getConfig();
 
-var Pushbullet = function(done) {
-    _.bindAll(this);
+const Pushbullet = function(done) {
+  _.bindAll(this);
 
-    this.pusher;
-    this.price = 'N/A';
+  this.currency = config.watch.currency;
+  this.asset = config.watch.asset;
+  this.exchange = config.watch.exchange;
+  this.strategy = config.tradingAdvisor.method;
 
-    this.done = done;
-    this.setup();
+  this.title = config.pushbullet.tag;
+  this.digitsCurrency = config.pushbullet.digitsCurrency;
+  this.digitsAsset = config.pushbullet.digitsAsset;
+
+  this.dates = {
+    start: false,
+    end: false
+  }
+  
+  this.startPrice = 0;
+  this.endPrice = 0;
+  
+  this.trades = 0;
+  
+  this.roundTrip = {
+    entry: false,
+    exit: false
+  }
+
+  this.price = 0;
+  this.prevAction = 'none';
+
+  this.pusher;
+  this.done = done;
+  this.setup();
 };
 
 Pushbullet.prototype.setup = function(done){
-
-    var setupPushBullet = function (err, result) {
-        if(pushbulletConfig.sendMessageOnStart){
-            var title = pushbulletConfig.tag;
-            var exchange = config.watch.exchange;
-            var currency = config.watch.currency;
-            var asset = config.watch.asset;
-            var body = "Gekko has started, Ive started watching "
-                +exchange
-                +" "
-                +currency
-                +" "
-                +asset
-                +" I'll let you know when I got some advice";
-            this.mail(title, body);
-        }else{
-            log.debug('Skipping Send message on startup')
-        }
-    };
-    setupPushBullet.call(this)
+  var setupPushBullet = function (err, result) {
+    if (config.pushbullet.startMessage) {
+      var subject = this.title + ' Started!'
+      var body = "Gekko has started and is trading LIVE on the " + this.exchange.toUpperCase() + " " + this.asset + "/" + this.currency + ' market using the strategy [' + this.strategy + ']';
```

Comment:
We do not know for sure whether Gekko is "trading LIVE", it might be a paper trader.

Reply:
Removed the word "LIVE" from the message